crates/ra_ide_api/Cargo.toml

```diff
@@ -27,6 +27,7 @@ ra_prof = { path = "../ra_prof" }
 hir = { path = "../ra_hir", package = "ra_hir" }
 test_utils = { path = "../test_utils" }
 ra_assists = { path = "../ra_assists" }
+text_unit = { version = "0.1.6", features = ["serde"] }
```

Comment:
All types of `text_unit` are reexported in `ra_syntax`, so there's no need to depend on `text_unit` explicitelly